StRoot/StBFChain/BigFullChain.h

```diff
@@ -1949,6 +1954,7 @@ Bfc_st BFC[] = { // standard chains
   {"PicoVtxVpdOrDefault","","","-PicoVtxDefault"    ,"","","pico Vtx cut on Tof and VPD or default",kFALSE},
   {"PicoVtxFXT"     ,"","","-PicoVtxDefault"    ,"" ,"","pico Vtx constraint on FXT [198,202] mode",kFALSE},
   {"PicoVtxMtd"     ,"","","-PicoVtxDefault"             ,"" ,"","pico Vtx using MTD matching mode",kFALSE},
+  {"PicoVtxless"    ,"","","-PicoVtxDefault"             ,"" ,"","pico Vtx NOT required for FWD"   ,kFALSE},
```

Comment:
```suggestion
  {"PicoVtxless"    ,"","","-PicoVtxDefault"                ,"" ,"","pico Vtx NOT required for FWD",kFALSE},
```

Reply:
Actually, don't we need a concurrent change in `StBFChain.cxx` to pass this along to StPicoDstMaker?

    else if ( GetOption("PicoVtxless"))      mk->SetAttr("PicoVtxMode", "PicoVtxless");

...near line 698 of StBFChain.cxx.

Reply:
Implemented in #639